### PR TITLE
Replace absl by python argparse from stdlib

### LINKED ISSUE #5: Replace absl-py by python stdlib and argparse
Hi!

I would to propose to you to reduce your project dependencies by replacing absl-py (gflag) by [argparse](https://docs.python.org/3/library/argparse.html) who is part of the python standard library.

Argparse is a great module and it can make the job efficiently without using a third dependency just for parsing script arguments.

Do you agree?

If you are ok with this I can easily implement it.

Give me your feedback please :)

### PR DESCRIPTION
**Pull request by 4383**
Hi,

# Overview
- remove dependency (absl)
- replace absl by python standard library argparse 
- remove unused import (os).
- update help

## Issues
Fix #5 

## Usage
```shell
$ git clone https://github.com/4383/anishot --single-branch -b absl_to_argparse
$ cd anishot
$ pip install pbr
$ python setup.py develop
$ anishot ~/screenshot.png /tmp/output.gif 486
Done! Generated file will be available at /tmp/output.gif
$ ls /tmp
output.py
$ anishot --help
usage: anishot [-h] [-p PAD] [-m MAXSPEED] [-s [STOPS [STOPS ...]]]
               [--zoom-steps ZOOM_STEPS] [--start-scale START_SCALE]
               [--zoom-to ZOOM_TO] [--shadow-size SHADOW_SIZE]
               [--rgb-outline RGB_OUTLINE] [--rgb-background RGB_BACKGROUND]
               [--rgb-shadow RGB_SHADOW] [--rgb-window RGB_WINDOW]
               input output height

Animates a long screenshot into a GIF

positional arguments:
  input                 Input screenshot image
  output                Output animated GIF
  height                Window height

optional arguments:
  -h, --help            show this help message and exit
  -p PAD, --pad PAD     Padding on sides
  -m MAXSPEED, --maxspeed MAXSPEED
                        Max speed on scroll px/frame
  -s [STOPS [STOPS ...]], --stops [STOPS [STOPS ...]]
                        Max speed on scroll px/frame
  --zoom-steps ZOOM_STEPS
                        Number of steps on initial zoom in
  --start-scale START_SCALE
                        Start scale
  --zoom-to ZOOM_TO     Point to zoom to
  --shadow-size SHADOW_SIZE
                        Shadow size
  --rgb-outline RGB_OUTLINE
                        Screenshot outline color
  --rgb-background RGB_BACKGROUND
                        Background color
  --rgb-shadow RGB_SHADOW
                        Screenshot shadow color
  --rgb-window RGB_WINDOW
                        Window outline color
```

### CODE REVIEW

File: anishot/__main__.py [hunk 1]
```diff
@@ -3,46 +3,65 @@
...
 
 
 def scale_image(image, scale):
-    w, h = int(image.width * scale + .5), int(image.height * scale + .5)
-    return image.resize((w, h), Image.LANCZOS)
+    try:
+        w, h = int(image.width * scale + .5), int(image.height * scale + .5)
+        return image.resize((w, h), Image.LANCZOS)
+    except ValueError:
+        pass
```
Comment (sergey48k) [inline anishot/__main__.py]: Hm. Why would ValueError arise here, and why would you ignore it?

**Review by sergey48k — APPROVED** [review]
Nice change, thank you. I am approving but see the comment re. ValueError. Looks like a forgotten debug code.

File: anishot/__main__.py (hunk 1 above)
Comment (4383) [reply anishot/__main__.py]: When the execution pass through this code at the last iteration width and height are equal to 0 so image resizing raise a ValueError exception due to these values.
I'm not really sure it's a good fix but the problem doesn't occur with this fix.

**Comment by 4383** [comment]
I think it's most preferable to merge this pull request before merging #6.
When this pull request will be merged I need to update code from #6 to remove absl dependency.
This process avoid merge conflict between these 2 pulls requests

**Comment by 4383** [comment]
Concerning the `ValueError` I have removed it because I've already manage this case with the following code (line 92):
```python
scaled = scale_image(image, scale)
if not scaled:
    continue
```

So, yes, we can saying it's an unused debug code :)

**Pull request merged**